backend/.env.example

```
@@ -0,0 +1,5 @@
+BETTER_AUTH_SECRET=        # openssl rand -base64 32
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)